cli/src/command.rs

```diff
@@ -6,8 +8,16 @@ pub enum Command {
     ExecuteFromFile(String),
 }
 
+#[derive(ThisError, Debug, PartialEq)]
+pub enum CommandError {
+    #[error("should specify table")]
+    LackOfTable,
+    #[error("command not supported")]
+    NotSupported,
+}
```

Comment:
Good 👍

Reply:
Do you think it is proper even if we don't use the error messges?
I still doubt if there is another way to distinguish those two cases without using Error

Reply:
Since they're in different branches, you might be able to tell them apart, but what's clear is that this looks good.